### PR TITLE
#163052943 Implement Feedback for UI templates

### PR DESCRIPTION
**Pull request by dayoaj**
#### What does this PR do?
Add Implementations for feedback to UI
#### Description of Task to be completed?
Add the Following Features:
- Header
  - Adjust Logo Text to Side
  - Adjust Nav Bar to be aligned and add proper padding
- Body
  - Add Uniform Background color of White
  -  Change Headline font Style to sans-serif
- Footer
  - Align Text to be well centered 
- Forms
  - Remove border radius and shadow
- Properly Align items in Meetup, Questions and Comments Page
- Rearrange Profile so all items can be centered and well spaced.
- Adjust Dashboard in Admin Page to be more centered and highlight on Hover
- Add date to upcoming meetup page

#### How should this be manually tested?

- Clone the project repo
- Open _index.html_ page on a browser

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163052943

#### Screenshots (if appropriate)
align-question-textbox
centered-profile
home-page
meetup-list
aligned-table

### CODE REVIEW

File: UI/addmeetup.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
                         <nav>
                             <ul>
                                 <li><a class="header-link" href="index.html">Home</a></li>
-                                <li><a class="header-link" href="#">About</a></li>
+
```
Comment (chukwuemekachm) [inline UI/addmeetup.html]: Please use two character spacing for this file.

File: UI/forms.css [hunk 2]
```diff
@@ -5,10 +5,17 @@
 .base {
 
 } */
+
 .signup-wrap {
```
Comment (chukwuemekachm) [inline UI/forms.css:9]: Please use two character spacing for this file.

File: UI/meetup-details.css [hunk 3]
```diff
@@ -1,9 +1,11 @@
 .meetup-details-body {
     margin: 2em auto;
```
Comment (chukwuemekachm) [inline UI/meetup-details.css]: Please use two character spacing for this file.

File: UI/profile.css [hunk 4]
```diff
@@ -1,170 +1,181 @@
...
 }
 
 .profile-img-wrap {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
 }
```
Comment (chukwuemekachm) [inline UI/profile.css]: Please use two character spacing for this file.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Good job @dayoaj.
Please attend to the comments below.

File: UI/addmeetup.html (hunk 1 above)
Comment (dayoaj) [reply UI/addmeetup.html]: Noted. Thanks.

File: UI/forms.css (hunk 2 above)
Comment (dayoaj) [reply UI/forms.css:9]: Noted, Thanks for the feedback

File: UI/meetup-details.css (hunk 3 above)
Comment (dayoaj) [reply UI/meetup-details.css]: Noted, Thanks

File: UI/profile.css (hunk 4 above)
Comment (dayoaj) [reply UI/profile.css]: Thanks for the feedback

File: UI/forms.css (hunk 2 above)
Comment (dayoaj) [reply UI/forms.css:9]: I just added the changes to the file

File: UI/meetup-details.css [hunk 5]
```diff
@@ -101,43 +99,41 @@ h2.meetup-article-title {
...
 
-.container-question-button{
+.container-question-button {
     display: flex;
     justify-content: flex-end;
 }
 
-.question{
+.question {
     display: flex;
```
Comment (chukwuemekachm) [inline UI/meetup-details.css]: Please resolve this file as well.

Comment (dayoaj) [reply UI/meetup-details.css]: Oh! Sorry for the omission.

**Review by chukwuemekachm — APPROVED** [review]
Well done @dayoaj.

**Pull request merged**